rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.61"
```

Comment:
Why add this one?

Reply:
Its failing to compile with latest cargo. Another reason for #171.

```

Compiling mdbook v0.4.15 (/home/felix/.cargo/git/checkouts/mdbook-66041be698320ee6/e74fdb1)
error[E0597]: `local_ctx` does not live long enough
--> src/renderer/html_handlebars/helpers/navigation.rs:154:25
|
154 |             t.render(r, &local_ctx, &mut local_rc, out)
|                         ^^^^^^^^^^ borrowed value does not live long enough
155 |         })?;
|         -
|         |
|         `local_ctx` dropped here while still borrowed
|         borrow might be used here, when `local_rc` is dropped and runs the destructor for type `handlebars::RenderContext<'_, '_>`
|
= note: values in a scope are dropped in the opposite order they are defined

For more information about this error, try `rustc --explain E0597`.
```